chatez/src/pages/Chats/Components/ChatFeed/ChatFeed.tsx

```diff
@@ -20,14 +20,25 @@ const ChatFeed: React.FC = () => {
   // Authentication and user context
   const { currentUserAccessToken, userLoggedIn, currentUser } = useAuth();
   currentUserAccessToken ?? console.log(currentUserAccessToken);
-  const { currentFriend, loadMessages, setLoadMessages } = useChat();
+  const {
+    currentFriend,
+    loadMessages,
+    setLoadMessages,
+    addedOrRemovedFriendStatus,
+    setAddedOrRemovedFriendStatus,
+    addedOrDeletedFriend,
+    setAddedOrDeletedFriend,
+    setAddedOrDeleted,
+    addedOrDeleted,
+  } = useChat();
```

Comment:
These fields sound confusing, can this be simplified?

Reply:
it can if i spend some time on it. ill do it tonight